sky/engine/core/dart/compositing.dart

```diff
@@ -4,31 +4,106 @@
+  ///
+  /// Rasterization outside the given rounded rectangle is discarded.
+  ///
+  /// See [pop] for details about the operation stack.
   void pushClipRRect(RRect rrect) native "SceneBuilder_pushClipRRect";
+
+  /// Pushes a path clip operation onto the operation stack.
+  ///
+  /// Rasterization outside the given path is discarded.
+  ///
+  /// See [pop] for details about the operation stack.
   void pushClipPath(Path path) native "SceneBuilder_pushClipPath";
+
+  /// Pushes an opacity operation onto the operation stack.
+  ///
+  /// The given alpha value is blended into the alpha value of the objects'
+  /// raterization. An alpha value of 0 makes the objects entirely invisible. An
+  /// alpha value of 255 has no effect (i.e., the objects retain the current
+  /// opacity).
+  ///
+  /// See [pop] for details about the operation stack.
   void pushOpacity(int alpha) native "SceneBuilder_pushOpacity";
+
+  /// Pushes a color filter operation onto the operation stack.
+  ///
+  /// The given color is applied to the objects' rasterization using the given
+  /// transfer mode.
+  ///
+  /// See [pop] for details about the operation stack.
   void pushColorFilter(Color color, TransferMode transferMode) native "SceneBuilder_pushColorFilter";
+
+  /// Pushes a shader mask operation onto the operation stack.
+  ///
+  /// The given shader is applied to the object's rasterization in the given
+  /// rectangle using the given transfer mode.
+  ///
+  /// See [pop] for details about the operation stack.
   void pushShaderMask(Shader shader, Rect maskRect, TransferMode transferMode) native "SceneBuilder_pushShaderMask";
 
+  /// Ends the effect of the most recently pushed operation.
+  ///
+  /// Internally the scene builder maintains a stack of operations. Each of the
+  /// operations in the stack applies to each of the objects added to the scene.
+  /// Calling this function removes the most recently added operation from the
+  /// stack.
   void pop() native "SceneBuilder_pop";
 
+  /// Adds an object to the scene that displays performance statistics.
+  ///
+  /// Useful during development to assess the performance of the application.
+  /// The enabledOptions controls which statistics are displayed. The bounds
+  /// controls where the statistics are displayed.
   void addPerformanceOverlay(int enabledOptions, Rect bounds) native "SceneBuilder_addPerformanceOverlay";
+
+  /// Adds a picture to the scene.
+  ///
+  /// The picture is rasterized at the given offset.
   void addPicture(Offset offset, Picture picture) native "SceneBuilder_addPicture";
+
+  /// (mojo-only) Adds a scene rendered by another app to the scene for this app.
   void addChildScene(Offset offset,
                      int physicalWidth,
                      int physicalHeight,
                      int sceneToken) native "SceneBuilder_addChildScene";
+
+  /// Sets a threshold after which additional debugging information should be recorded.
```

Comment:
If that's the case, that's probably worth saying. Maybe tell them to contact flutter-dev if they think they need this, so that we can figure out how to improve it.